source/common/tls/context_config_impl.cc

```diff
@@ -516,6 +537,13 @@ Ssl::ServerContextConfig::OcspStaplePolicy ServerContextConfigImpl::ocspStaplePo
   PANIC_DUE_TO_CORRUPT_ENUM;
 }
 
+Ssl::TlsCertificateSelectorFactoryCb ServerContextConfigImpl::createTlsCertificateSelector() const {
```

Comment:
I don't think this function is needed; the selector will always be configured in the constructor